app/src/main/java/ch/epfl/sdp/appart/scrolling/ad/Ad.java

```diff
@@ -0,0 +1,63 @@
+package ch.epfl.sdp.appart.scrolling.ad;
+
+import java.util.List;
+
+public class Ad {
+
+    private String title;
+    private String price;
+    private String address;
+    private String advertiserId;
+    private String description;
+    private List<String> photosRefs;
+    private boolean hasVTour;
+
+    public Ad(String title, String price, String address,
+              String advertiserId, String description, List<String> photosRefs, boolean hasVTour) {
+        if (title == null || price == null || address == null)
+            throw new IllegalArgumentException("An argument is null!");
+        if (advertiserId == null || description == null || photosRefs == null)
+            throw new IllegalArgumentException("An argument is null!");
```

Comment:
Why the double if?

Reply:
It's because code climate was telling the condition was too complex when putting everything in the same one

Reply:
Yes, got it!